src/components/Board/Navbar/Navbar.css

```diff
@@ -7,6 +7,11 @@
   color: #fff;
 }
 
+.Navbar__dark {
```

Comment:
the class name should be `.Navbar--dark`.
We're using [B.E.M. class naming convention](https://css-tricks.com/bem-101/) - and dark is a modifier.